Broken installation because of Cython/Python dependencies

This formula depends both on Python@3.10 and Cython, but the current Cython formula version requires Python@3.11.
Because of this, installation fails with:

```
brew install ceph-client --verbose --debug
...
==> Fetching dependencies for mulbc/ceph-client/ceph-client: cython
==> Fetching cython
==> Downloading https://ghcr.io/v2/homebrew/core/cython/manifests/0.29.33
...
==> Fetching mulbc/ceph-client/ceph-client
...
==> Verifying checksum for 'b368830312f6eb917b2570de1804003eb370a49ca1887c698544c64b8c8c8313--ceph-17.2.5.tar.gz'
==> Installing ceph-client from mulbc/ceph-client
==> Installing dependencies for mulbc/ceph-client/ceph-client: cython
==> Installing mulbc/ceph-client/ceph-client dependency: cython
...
==> Installing mulbc/ceph-client/ceph-client
...
==> Patching
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:239:in `mkdir'
Errno::EPERM: Operation not permitted @ dir_s_mkdir - /usr/local/opt/cython/libexec/lib/python3.10
...
```

Stack trace:
```
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:239:in `fu_mkdir'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:217:in `block (2 levels) in mkdir_p'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:215:in `reverse_each'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:215:in `block in mkdir_p'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:200:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/fileutils.rb:200:in `mkdir_p'
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/pathname.rb:578:in `mkpath'
/usr/local/Homebrew/Library/Homebrew/extend/ENV/shared.rb:133:in `prepend_create_path'
/usr/local/Homebrew/Library/Taps/mulbc/homebrew-ceph-client/ceph-client.rb:58:in `install'
```

That is because current Cython formula installed just the Python@3.11 libs in that path:
```
ls -1 /usr/local/opt/cython/libexec/lib
python3.11
```

I managed to pass this error by changing `python3.10` to `python3.11` in the `ceph-client.rb ` file but I'm encountering new errors about arch incompatibility :
> ps: I'm on Mac M1 
```log

$ brew install homebrew-ceph-client/ceph-client.rb                                                                                                                                                                                                         

Error: Failed to load cask: homebrew-ceph-client/ceph-client.rb
Cask 'ceph-client' is unreadable: wrong constant name #<Class:0x0000000113055438>
Warning: Treating homebrew-ceph-client/ceph-client.rb as a formula.
==> Fetching ceph-client
==> Downloading https://files.pythonhosted.org/packages/cb/7d/7e6bc4bd4abc49e9f4f5c4773bb43d1615e4b476d108d1b527318b9c6521/prettytable-3.2.0.tar.gz
Already downloaded: /Users/ilyes.elajroud/Library/Caches/Homebrew/downloads/4c52d4026151aa261d150292e430951006c98d7d72a89034ecea460b2f2c111b--prettytable-3.2.0.tar.gz
==> Downloading https://files.pythonhosted.org/packages/36/2b/61d51a2c4f25ef062ae3f74576b01638bebad5e045f747ff12643df63844/PyYAML-6.0.tar.gz
Already downloaded: /Users/ilyes.elajroud/Library/Caches/Homebrew/downloads/83f6bef3a0a4fd3d9559a85acb4ab73b0e4544f8542e67bf54929e86653356e9--PyYAML-6.0.tar.gz
==> Downloading https://files.pythonhosted.org/packages/89/38/459b727c381504f361832b9e5ace19966de1a235d73cdbdea91c771a1155/wcwidth-0.2.5.tar.gz
Already downloaded: /Users/ilyes.elajroud/Library/Caches/Homebrew/downloads/bc4d16c76a1cb1eecc9775864d059d1cfb00289c163bc86440fcd6eef95aa2d8--wcwidth-0.2.5.tar.gz
==> Downloading https://download.ceph.com/tarballs/ceph-17.2.5.tar.gz
Already downloaded: /Users/ilyes.elajroud/Library/Caches/Homebrew/downloads/b368830312f6eb917b2570de1804003eb370a49ca1887c698544c64b8c8c8313--ceph-17.2.5.tar.gz
==> Patching
==> python3 -c import setuptools... --no-user-cfg install --prefix=/opt/homebrew/Cellar/ceph-client/17.2.5_1/libexec/vendor --install-scripts=/opt/homebrew/Cellar/ceph-client/17.2.5_1/libexec/vendor/bin --install-lib=/opt/homebrew/Cellar/ceph-client/17.2.5_1/libexec/ven
==> python3 -c import setuptools... --no-user-cfg install --prefix=/opt/homebrew/Cellar/ceph-client/17.2.5_1/libexec/vendor --install-scripts=/opt/homebrew/Cellar/ceph-client/17.2.5_1/libexec/vendor/bin --install-lib=/opt/homebrew/Cellar/ceph-client/17.2.5_1/libexec/ven
==> python3 -c import setuptools... --no-user-cfg install --prefix=/opt/homebrew/Cellar/ceph-client/17.2.5_1/libexec/vendor --install-scripts=/opt/homebrew/Cellar/ceph-client/17.2.5_1/libexec/vendor/bin --install-lib=/opt/homebrew/Cellar/ceph-client/17.2.5_1/libexec/ven
==> cmake -G Ninja .. -DDIAGNOSTICS_COLOR=always -DOPENSSL_ROOT_DIR=/opt/homebrew/opt/openssl@3 -DWITH_BABELTRACE=OFF -DWITH_BLUESTORE=OFF -DWITH_CCACHE=OFF -DWITH_CEPHFS=OFF -DWITH_KRBD=OFF -DWITH_LIBCEPHFS=ON -DWITH_LTTNG=OFF -DWITH_LZ4=OFF -DWITH_MANPAGE=ON -DWITH_MG
==> ninja rados rbd cephfs ceph-conf ceph-fuse manpages cython_rados cython_rbd
Last 15 lines from /Users/ilyes.elajroud/Library/Logs/Homebrew/ceph-client/05.ninja:
    src/CMakeFiles/common-objs.dir/mon/MonCap.cc.o
duplicate symbol 'boost::phoenix::placeholders::uarg1' in:
    src/common/CMakeFiles/common-common-objs.dir/ConfUtils.cc.o
    src/CMakeFiles/common-objs.dir/mon/MonCap.cc.o
duplicate symbol 'boost::phoenix::placeholders::uarg10' in:
    src/common/CMakeFiles/common-common-objs.dir/ConfUtils.cc.o
    src/CMakeFiles/common-objs.dir/mon/MonCap.cc.o
ld: 10 duplicate symbols for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
[268/565] Linking CXX static library lib/libcommon.a
[269/565] Building CXX object src/neorados/CMakeFiles/neorados_api_obj.dir/RADOS.cc.o
[270/565] Building CXX object src/tools/CMakeFiles/rados.dir/RadosDump.cc.o
[271/565] Building CXX object src/tools/CMakeFiles/rados.dir/rados/RadosImport.cc.o
[272/565] Building CXX object src/tools/CMakeFiles/rados.dir/rados/rados.cc.o
ninja: build stopped: subcommand failed.

```

I tried switching to Python 3.11 on amd64 (Intel) arch:

```
==> python3 -c import setuptools... --no-user-cfg install --prefix=/usr/local/Cellar/ceph-client/17.2.5_1/libexec/vendor --install-scripts=/usr/local/Cellar/ceph-client/17.2.5_1/libexec/vendor/bin --install-lib=/usr/local/Cellar/ceph-client/17.2.5_1/libexec/vendor/lib/python3.11/site-packages --single-version-externally-managed --record=installed.txt
/usr/local/lib/python3.11/site-packages/setuptools/config/setupcfg.py:508: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
  warnings.warn(msg, warning_class)
running install
/usr/local/lib/python3.11/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
running build
running build_py
creating build
creating build/lib
creating build/lib/wcwidth
copying wcwidth/wcwidth.py -> build/lib/wcwidth
copying wcwidth/table_zero.py -> build/lib/wcwidth
copying wcwidth/__init__.py -> build/lib/wcwidth
copying wcwidth/table_wide.py -> build/lib/wcwidth
copying wcwidth/unicode_versions.py -> build/lib/wcwidth
warning: build_py: byte-compiling is disabled, skipping.

running install_lib
creating /usr/local/Cellar/ceph-client/17.2.5_1/libexec/vendor/lib/python3.11/site-packages/wcwidth
copying build/lib/wcwidth/wcwidth.py -> /usr/local/Cellar/ceph-client/17.2.5_1/libexec/vendor/lib/python3.11/site-packages/wcwidth
copying build/lib/wcwidth/table_zero.py -> /usr/local/Cellar/ceph-client/17.2.5_1/libexec/vendor/lib/python3.11/site-packages/wcwidth
copying build/lib/wcwidth/__init__.py -> /usr/local/Cellar/ceph-client/17.2.5_1/libexec/vendor/lib/python3.11/site-packages/wcwidth
copying build/lib/wcwidth/table_wide.py -> /usr/local/Cellar/ceph-client/17.2.5_1/libexec/vendor/lib/python3.11/site-packages/wcwidth
copying build/lib/wcwidth/unicode_versions.py -> /usr/local/Cellar/ceph-client/17.2.5_1/libexec/vendor/lib/python3.11/site-packages/wcwidth
warning: install_lib: byte-compiling is disabled, skipping.

running install_egg_info
running egg_info
writing wcwidth.egg-info/PKG-INFO
writing dependency_links to wcwidth.egg-info/dependency_links.txt
writing requirements to wcwidth.egg-info/requires.txt
writing top-level names to wcwidth.egg-info/top_level.txt
reading manifest file 'wcwidth.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
writing manifest file 'wcwidth.egg-info/SOURCES.txt'
Copying wcwidth.egg-info to /usr/local/Cellar/ceph-client/17.2.5_1/libexec/vendor/lib/python3.11/site-packages/wcwidth-0.2.5-py3.11.egg-info
running install_scripts
writing list of installed files to 'installed.txt'
==> cmake -G Ninja .. -DDIAGNOSTICS_COLOR=always -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl@3 -DWITH_BABELTRACE=OFF -DWITH_BLUESTORE=OFF -DWITH_CCACHE=OFF -DWITH_CEPHFS=OFF -DWITH_KRBD=OFF -DWITH_LIBCEPHFS=ON -DWITH_LTTNG=OFF -DWITH_LZ4=OFF -DWITH_MANPAGE=ON -DWITH_MGR=OFF -DWITH_MGR_DASHBOARD_FRONTEND=OFF -DWITH_RADOSGW=OFF -DWITH_RDMA=OFF -DWITH_SPDK=OFF -DWITH_SYSTEM_BOOST=ON -DWITH_SYSTEMD=OFF -DWITH_TESTS=OFF -DWITH_XFS=OFF -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/ceph-client/17.2.5_1 -DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev -DBUILD_TESTING=OFF -DCMAKE_OSX_SYSROOT=/Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk
-- The CXX compiler identification is AppleClang 14.0.0.14000029
-- The C compiler identification is AppleClang 14.0.0.14000029
-- The ASM compiler identification is Clang with GNU-like command-line
-- Found assembler: /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Looking for memset_s
-- Looking for memset_s - found
-- Looking for fallocate
-- Looking for fallocate - not found
-- Looking for posix_fadvise
-- Looking for posix_fadvise - not found
-- Looking for posix_fallocate
-- Looking for posix_fallocate - not found
-- Looking for syncfs
-- Looking for syncfs - not found
-- Looking for sync_file_range
-- Looking for sync_file_range - not found
-- Looking for pwritev
-- Looking for pwritev - found
-- Looking for splice
-- Looking for splice - not found
-- Looking for getgrouplist
-- Looking for getgrouplist - found
-- Looking for strerror_r
-- Looking for strerror_r - found
-- Looking for name_to_handle_at
-- Looking for name_to_handle_at - not found
-- Looking for pipe2
-- Looking for pipe2 - not found
-- Looking for accept4
-- Looking for accept4 - not found
-- Looking for sigdescr_np
-- Looking for sigdescr_np - not found
-- Looking for pthread_spin_init
-- Looking for pthread_spin_init - not found
-- Looking for pthread_set_name_np
-- Looking for pthread_set_name_np - not found
-- Looking for pthread_get_name_np
-- Looking for pthread_get_name_np - not found
-- Looking for pthread_setname_np
-- Looking for pthread_setname_np - found
-- Looking for pthread_getname_np
-- Looking for pthread_getname_np - found
-- Looking for pthread_rwlockattr_setkind_np
-- Looking for pthread_rwlockattr_setkind_np - not found
-- Looking for eventfd
-- Looking for eventfd - not found
-- Looking for getprogname
-- Looking for getprogname - found
-- Looking for gettid
-- Looking for gettid - not found
-- Looking for include file linux/types.h
-- Looking for include file linux/types.h - not found
-- Looking for include file linux/version.h
-- Looking for include file linux/version.h - not found
-- Looking for include file arpa/nameser_compat.h
-- Looking for include file arpa/nameser_compat.h - found
-- Looking for include file sys/mount.h
-- Looking for include file sys/mount.h - found
-- Looking for include file sys/param.h
-- Looking for include file sys/param.h - found
-- Looking for include file sys/types.h
-- Looking for include file sys/types.h - found
-- Looking for include file sys/vfs.h
-- Looking for include file sys/vfs.h - not found
-- Looking for include file sys/prctl.h
-- Looking for include file sys/prctl.h - not found
-- Looking for include file execinfo.h
-- Looking for include file execinfo.h - found
-- Looking for include file valgrind/helgrind.h
-- Looking for include file valgrind/helgrind.h - not found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of __u8
-- Check size of __u8 - failed
-- Check size of __u16
-- Check size of __u16 - failed
-- Check size of __u32
-- Check size of __u32 - failed
-- Check size of __u64
-- Check size of __u64 - failed
-- Check size of __s8
-- Check size of __s8 - failed
-- Check size of __s16
-- Check size of __s16 - failed
-- Check size of __s32
-- Check size of __s32 - failed
-- Check size of __s64
-- Check size of __s64 - failed
-- Check size of in_addr_t
-- Check size of in_addr_t - failed
-- Looking for _POSIX_TIMERS
-- Looking for _POSIX_TIMERS - not found
-- Looking for res_nquery
-- Looking for res_nquery - found
-- Looking for F_SETPIPE_SZ
-- Looking for F_SETPIPE_SZ - not found
-- Looking for __func__
-- Looking for __func__ - found
-- Looking for __PRETTY_FUNCTION__
-- Looking for __PRETTY_FUNCTION__ - found
-- Looking for getentropy
-- Looking for getentropy - not found
-- Performing Test STRERROR_R_CHAR_P
-- Performing Test STRERROR_R_CHAR_P - Failed
-- Performing Test HAVE_STAT_ST_MTIM_TV_NSEC
-- Performing Test HAVE_STAT_ST_MTIM_TV_NSEC - Failed
-- Performing Test HAVE_STAT_ST_MTIMESPEC_TV_NSEC
-- Performing Test HAVE_STAT_ST_MTIMESPEC_TV_NSEC - Success
-- Performing Test HAVE_UNALIGNED_ACCESS
-- Performing Test HAVE_UNALIGNED_ACCESS - Success
-- Performing Test HAVE_ATTR_SYMVER
-- Performing Test HAVE_ATTR_SYMVER - Failed
-- Performing Test HAVE_ASM_SYMVER
-- Performing Test HAVE_ASM_SYMVER - Failed
-- Performing Test HAVE_CXX11_ATOMIC
-- Performing Test HAVE_CXX11_ATOMIC - Success
-- Looking for backtrace
-- Looking for backtrace - found
-- backtrace facility detected in default set of libraries
-- Found Backtrace: /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/include
-- Found OpenLDAP: /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/include
-- Could NOT find FUSE (missing: FUSE_LIBRARIES FUSE_INCLUDE_DIR) (found version ".")
-- Found SQLite3: /usr/local/opt/sqlite/lib/libsqlite3.dylib
-- Found snappy: /usr/local/lib/libsnappy.dylib
-- Found gperftools: /usr/local/include (found suitable version "2.10", minimum required is "2.6.2")
-- Found CURL: /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/lib/libcurl.tbd (found version "7.85.0")
-- Looking for curl_multi_wait
-- Looking for curl_multi_wait - found
-- Found OpenSSL: /usr/local/opt/openssl@3/lib/libcrypto.dylib (found version "3.0.7")
-- Found Python3: /usr/local/bin/python3.10 (found suitable exact version "3.10.9") found components: Interpreter Development
-- Found ZLIB: /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/lib/libz.tbd (found version "1.2.11")
-- Found Boost: /usr/local/lib/cmake/Boost-1.81.0/BoostConfig.cmake (found suitable version "1.81.0", minimum required is "1.73") found components: atomic chrono thread system regex random program_options date_time iostreams context coroutine
-- Found Threads: TRUE
-- Found StdFilesystem: included by standard library
-- Ceph/.git directory not found, parsing /tmp/ceph-client-20230201-36011-fpte6w/ceph-17.2.5/src/.git_version for CEPH_GIT_VER and CEPH_GIT_NICE_VER
-- Performing Test COMPILER_SUPPORTS_PESSIMIZING_MOVE
-- Performing Test COMPILER_SUPPORTS_PESSIMIZING_MOVE - Success
-- Performing Test COMPILER_SUPPORTS_REDUNDANT_MOVE
-- Performing Test COMPILER_SUPPORTS_REDUNDANT_MOVE - Success
-- Performing Test HAS_GLIBCXX_ASSERTIONS
-- Performing Test HAS_GLIBCXX_ASSERTIONS - Success
-- Performing Test HAVE_INTEL_SSE
-- Performing Test HAVE_INTEL_SSE - Success
-- Performing Test HAVE_INTEL_SSE2
-- Performing Test HAVE_INTEL_SSE2 - Success
-- Performing Test HAVE_INTEL_SSE3
-- Performing Test HAVE_INTEL_SSE3 - Success
-- Performing Test HAVE_INTEL_SSSE3
-- Performing Test HAVE_INTEL_SSSE3 - Success
-- Performing Test HAVE_INTEL_PCLMUL
-- Performing Test HAVE_INTEL_PCLMUL - Success
-- Performing Test HAVE_INTEL_SSE4_1
-- Performing Test HAVE_INTEL_SSE4_1 - Success
-- Performing Test HAVE_INTEL_SSE4_2
-- Performing Test HAVE_INTEL_SSE4_2 - Success
-- Could NOT find nasm
-- Performing Test HAVE_STDLIB_MAP_SPLICING
-- Performing Test HAVE_STDLIB_MAP_SPLICING - Success
-- Performing Test COMPILER_SUPPORTS_DIAGNOSTICS_COLOR
-- Performing Test COMPILER_SUPPORTS_DIAGNOSTICS_COLOR - Success
-- Could not find fmt, will build it
-- Module support is disabled.
-- Version: 7.1.4
-- Build type: Release
-- CXX_STANDARD: 17
-- Performing Test has_std_17_flag
-- Performing Test has_std_17_flag - Success
-- Performing Test has_std_1z_flag
-- Performing Test has_std_1z_flag - Success
-- Performing Test SUPPORTS_USER_DEFINED_LITERALS
-- Performing Test SUPPORTS_USER_DEFINED_LITERALS - Success
-- Performing Test FMT_HAS_VARIANT
-- Performing Test FMT_HAS_VARIANT - Success
-- Required features: cxx_variadic_templates
-- Performing Test HAS_NULLPTR_WARNING
-- Performing Test HAS_NULLPTR_WARNING - Success
-- Looking for strtod_l
-- Looking for strtod_l - found
-- Check size of suseconds_t
-- Check size of suseconds_t - done
-- Performing Test HAS_VTA
-- Performing Test HAS_VTA - Failed
-- Performing Test HAS_WARNING_STRINGOP_TRUNCATION
-- Performing Test HAS_WARNING_STRINGOP_TRUNCATION - Failed
-- Performing Test HAS_WARNING_DEPRECATED_COPY
-- Performing Test HAS_WARNING_DEPRECATED_COPY - Success
-- Performing Test HAS_WARNING_PESSIMIZING_MOVE
-- Performing Test HAS_WARNING_PESSIMIZING_MOVE - Success
-- Found Cython: 0.29
-- Looking for wsyncup in /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/lib/libcurses.tbd
-- Looking for wsyncup in /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/lib/libcurses.tbd - found
-- Looking for cbreak in /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/lib/libncurses.tbd
-- Looking for cbreak in /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/lib/libncurses.tbd - found
-- Looking for nodelay in /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/lib/libncurses.tbd
-- Looking for nodelay in /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/lib/libncurses.tbd - found
-- Found Curses: /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/lib/libncurses.tbd
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)
-- exclude following files under src: *.js;*.css;.tox;python-common/build;erasure-code/jerasure/jerasure;erasure-code/jerasure/gf-complete;rocksdb;googletest;spdk;xxHash;isa-l;zstd;crypto/isa-l/isa-l_crypto;blkin;rapidjson;dmclock;seastar;fmt;c-ares;spawn;pybind/mgr/rook/rook-client-python;s3select;libkmip;arrow;utf8proc
-- Configuring done
CMake Error at src/CMakeLists.txt:661 (target_link_libraries):
  Target "ceph-osd" links to:

    FUSE::FUSE

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at src/CMakeLists.txt:811 (target_link_libraries):
  Target "ceph-fuse" links to:

    FUSE::FUSE

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at src/os/CMakeLists.txt:83 (target_link_libraries):
  Target "os" links to:

    FUSE::FUSE

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at src/tools/CMakeLists.txt:62 (target_link_libraries):
  Target "ceph-objectstore-tool" links to:

    FUSE::FUSE

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



CMake Error at src/rbd_fuse/CMakeLists.txt:3 (target_link_libraries):
  Target "rbd-fuse" links to:

    FUSE::FUSE

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.



-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING


CMake Generate step failed.  Build files cannot be regenerated correctly.
/usr/local/Homebrew/Library/Homebrew/shims/shared/git --version
/usr/local/Homebrew/Library/Homebrew/shims/shared/curl --version
/usr/local/Homebrew/Library/Homebrew/ignorable.rb:29:in `block in raise'
BuildError: Failed executing: cmake -G Ninja .. -DDIAGNOSTICS_COLOR=always -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl@3 -DWITH_BABELTRACE=OFF -DWITH_BLUESTORE=OFF -DWITH_CCACHE=OFF -DWITH_CEPHFS=OFF -DWITH_KRBD=OFF -DWITH_LIBCEPHFS=ON -DWITH_LTTNG=OFF -DWITH_LZ4=OFF -DWITH_MANPAGE=ON -DWITH_MGR=OFF -DWITH_MGR_DASHBOARD_FRONTEND=OFF -DWITH_RADOSGW=OFF -DWITH_RDMA=OFF -DWITH_SPDK=OFF -DWITH_SYSTEM_BOOST=ON -DWITH_SYSTEMD=OFF -DWITH_TESTS=OFF -DWITH_XFS=OFF -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/ceph-client/17.2.5_1 -DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_VERBOSE_MAKEFILE=ON -Wno-dev -DBUILD_TESTING=OFF -DCMAKE_OSX_SYSROOT=/Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk
```

--- a/ceph-client.rb
+++ b/ceph-client.rb
@@ -20,7 +20,7 @@ class CephClient < Formula
   depends_on "leveldb" => :build
   depends_on "nss"
   depends_on "pkg-config" => :build
-  depends_on "python@3.10"
+  depends_on "python@3.11"
   depends_on "sphinx-doc" => :build
   depends_on "yasm"
   def caveats
@@ -77,6 +77,7 @@ class CephClient < Formula
       -DWITH_MANPAGE=ON
       -DWITH_MGR=OFF
       -DWITH_MGR_DASHBOARD_FRONTEND=OFF
+      -DWITH_PYTHON3=3.11
       -DWITH_RADOSGW=OFF
       -DWITH_RDMA=OFF
       -DWITH_SPDK=OFF
@@ -214,7 +215,7 @@ index 9d66ae979a6..eabf22bf174 100644
      set(ENV{CEPH_LIBDIR} \"${CMAKE_LIBRARY_OUTPUT_DIRECTORY}\")
 
 -    set(options --prefix=${CMAKE_INSTALL_PREFIX})
-+    set(options --prefix=${CMAKE_INSTALL_PREFIX} --install-lib=${CMAKE_INSTALL_PREFIX}/lib/python3.10/site-packages)
++    set(options --prefix=${CMAKE_INSTALL_PREFIX} --install-lib=${CMAKE_INSTALL_PREFIX}/lib/python3.11/site-packages)
      if(DEFINED ENV{DESTDIR})
        if(EXISTS /etc/debian_version)
          list(APPEND options --install-layout=deb)

